cart/views.py

```diff
@@ -1,3 +1,92 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.views     import View
+from django.http      import JsonResponse
+
+from .models   import Cart
+
+from utils import login_decorator
+
+class CartListView(View):
+
+    @login_decorator
+    def post(self,request):
+        try:
+            data = json.loads(request.body)
+            user = request.user
+
+            if data['count'] <= 0:
+                return JsonResponse({"message": "LESS_THAN_MINIMUM_QUANTITY"}, status=400) 
+
+            if Cart.objects.filter(user=user, ingredient_id=data['ingredient_id']).exists():
+                cart = Cart.objects.get(user=user, ingredient_id=data['ingredient_id'])
+                cart.count += data['count']
+                cart.save()
+                return JsonResponse({'message':'ADDITIONAL_SUCCESS'}, status=200) 
+                    
+            Cart.objects.create(
+                count         = data['count'],
+                ingredient_id = data['ingredient_id'],
+                user          = user
+                )
+            return JsonResponse({'message':'SUCCESS'}, status=201)
+        
+        except KeyError:
+            return JsonResponse({"message": "KEY_ERROR"}, status=400)
+        
+        except Cart.DoesNotExist:
+            return JsonResponse({'message': 'THIS_ACCOUNT_DOES_NOT_EXIST'}, status=400)
+    
+    @login_decorator
+    def get(self,request):
+        try:
+            users_ingredient_list = Cart.objects.filter(user=request.user)
+            cart_list = []
+
+            for cart in users_ingredient_list:
+                cart_info = {
+                    'id'       : cart.ingredient.id,
+                    'name'     : cart.ingredient.name,
+                    'price'    : cart.ingredient.price,
+                    'image_url': cart.ingredient.image_url,
+                    'count'    : cart.count
+                    }
+                cart_list.append(cart_info)
```

Comment:
변수가 verbose하지 않고 좀 더 간단하게 선언하면 가독성이 좋아질 것 같아요!
`Cart.objects.filter(user=request.user)`가 호출되고 반환되는 객체는 cart 인스턴스 입니다.

아래 코드도 한줄 더 줄여볼 수 있겠네요. cart info를 아래 여러곳에서 사용하는게 아니라면 바로 배열에 append하는게 더 좋을 것 같습니다.
```suggestion
            carts = Cart.objects.filter(user=request.user)
            
            cart_list = []

            for cart in carts:
                cart_list.append({
                    'id'       : cart.ingredient.id,
                    'name'     : cart.ingredient.name,
                    'price'    : cart.ingredient.price,
                    'image_url': cart.ingredient.image_url,
                    'count'    : cart.count
                })
```

Reply:
변수명 지을 때 반환되는 객체를 생각하면 되겠네여! 물론 그게 어렵지만 ㅎㅎ.. append는 좀 더 고민하고 작성했으면 피드백 주신 대로 작성할 수 있었을텐데 ㅠㅠ 앞으로 한번 더 생각하고 작성하겠습니다 ㅠㅠ!